guide/C/ch_invest.xml

```diff
@@ -971,56 +980,165 @@ Income
-      account will be used. If you wish to track commissions to the individual stock
-      an additional sub-account would be necessary.</para>
-
-      <para>Now for the transaction, on the first transaction line, enter the date of
-      the purchase (eg: Jan. 3 2005), a description (eg: Initial IBM Purchase),
-      Shares will be skipped (to be calculated), Price (eg: $96.60), and Buy ($5000).
-      You do not need to fill in the <guilabel>Shares</guilabel> column, as it will be
-      calculated for you. The next line in the split transaction will be
-      <emphasis>Expenses:Commissions</emphasis> and fill in Buy ($100).
-      The third line will be to transfer from <emphasis>Assets:Bank ABC:Savings</emphasis>,
-      $5100 to balance the transaction.
-      Your IBM Commodity view should now appear like this:</para>
-
-      <screenshot id="invest-setup-portfolio2">
-        <mediaobject>
-           <imageobject role="html">
-             <imagedata fileref="figures/invest_SetupPortfolio2.png" format="PNG"
-                        srccredit="Cristian Marchi" width="510px"></imagedata>
-           </imageobject>
-           <imageobject role="fo">
-             <imagedata fileref="figures/invest_SetupPortfolio2.png" format="PNG"
-                        srccredit="Cristian Marchi"></imagedata>
-           </imageobject>
+      <para>The main difference between setting up a new stock purchase
+        versus the setup for preexisting stocks as described in the previous
+        section is that instead of transferring the money used to purchase the
+        stock from the <emphasis>Equity:Opening Balances</emphasis> account, you
+        transfer from  your <emphasis>Assets:Bank ABC</emphasis> or <emphasis>
+        Assets:Brokerage Account</emphasis> account.
+      </para>
 
-          <textobject>
-            <phrase>Setup Current Portfolio</phrase>
-          </textobject>
+      <sect3 id="invest-buy-com">
+        <title>Handling Commission</title>
 
-          <caption><para>This image is of the transaction register of the IBM account
-          after the first stock account <quote>purchase</quote> with a commission.</para>
-          </caption>
-        </mediaobject>
-      </screenshot>
+        <para>
+          There are 2 alternate ways to handle the fees you pay to a broker to
+          buy or sell securities (for simplicity this document section will
+          refer to these as net pricing and gross pricing):
+        </para>
+
+        <itemizedlist>
+          <listitem><para><emphasis>Net Pricing:</emphasis> You enter a net
+            price (adjusted by commission) when buying and selling securities.
+            You do <emphasis>not</emphasis> also record commission in a specific
+            commission account in order to later claim it as a tax expense, as
+            this would be claiming commission twice. To use this method,
+            commission must be deductible in your jurisdiction (as it is in US
```

Comment:
We might suggest the users to ask their CPAs, which methods are allowed. But we might explain the effect: If you split the commissions and fees off, you can claim them as expense this year. If not they are captalized until you sell the shares again.
Another aspect might be to use the same methode as the broker did in his statement e.g. in countries with withholding tax.